A32NX/html_ui/Pages/A32NX_Core/A32NX_TransitionAltitude.js

```diff
@@ -0,0 +1,345 @@
+class A32NX_TransitionAltitude {
+    init() {
+        console.log('A32NX_TransitionAltitude init');
+        this.currentDeparture = "";
+        this.currentArrival = "";
+        this.offline = false;
+        this.checkStart = true;
+        this.checkStartSec = 0;
+        this.testAirport = "";
+        this.trySendRequest();
+    }
+
+    update(_deltaTime, _core) {
+        if (this.offline === true) {
+            this.offlineTransAlt();
+        }
+        this.tryCheckAPI(_deltaTime);
+    }
+
+    async trySendRequest() {
+        await NXApi.getAirport("KLAX")
+            .then((data) => {
+                this.testAirport = data.transAlt;
+            });
+    }
```

Comment:
**question**: Why do you use `async` and `await` here? What is your intended effect?

Reply:
Try send our api server, wait KLAX TA, if there is no response in 30sec, will use offline db